src/components/pip/pip.scss

```diff
@@ -47,9 +53,17 @@ $hideContainerGap: 5px;
     }
   }
 
+  .playerContainer {
+    position: relative;
+    border-radius: 8px;
+    overflow: hidden;
+    box-shadow: 0 0 40px 0 rgba(0, 0, 0, 0.2), 0 0 4px 0 rgba(0, 0, 0, 0.3);
+  }
+
   .innerButtons {
     flex-direction: row-reverse;
     border-radius: 8px;
+    z-index: 1;
```

Comment:
Don't use z-index, instead - replace the order of the container and the inner buttons
![image](https://user-images.githubusercontent.com/48884909/123310680-02acf280-d52f-11eb-9d0c-c8af9b1f5dbc.png)

Reply:
done